lib/tasks/invite_code.rake

```diff
@@ -1,16 +1,22 @@
-
+require 'base64'
+require 'securerandom'
 
 desc "Generate a batch of invite codes"
-task :generate_invites, [:n] => :environment do |task, args|
+task :generate_invites, [:n, :u] => :environment do |task, args|
```

Comment:
This is probably something left over from the git merge/rebase mess we cleaned up since yesterday... so the version with both :n and :u is correct and it should stay, but it's actually part of the multi-use invite codes, not this pull request. Not sure how exactly it ended up here... :neutral_face: but yes, it's what should stay.